test/AutoRest.TestServer.Tests/paging.cs

```diff
@@ -309,10 +309,10 @@ await foreach (var page in pageableAsync.AsPages())
             Assert.AreEqual(id, resultPage.Values.First().Properties.Id);
             Assert.AreEqual(product, resultPage.Values.First().Properties.Name);
             Assert.AreEqual("*&*#&$", resultPage.ContinuationToken);
-            Assert.ThrowsAsync<UriFormatException>(async () => await new PagingClient(ClientDiagnostics, pipeline, host).RestClient.GetMultiplePagesFailureNextPageAsync(resultPage.ContinuationToken));
+            Assert.ThrowsAsync<RequestFailedException>(async () => await new PagingClient(ClientDiagnostics, pipeline, host).RestClient.GetMultiplePagesFailureNextPageAsync(resultPage.ContinuationToken));
```

Comment:
Since we properly concatenate host + nextLink now, this fails with RequestFailedException (as it should).